sp/src/devtools/makefile_base_posix.mak

```diff
@@ -6,10 +6,10 @@ MAKEFILE_LINK:=$(THISFILE).link
 
 -include $(MAKEFILE_LINK)
 
-$(MAKEFILE_LINK): $(shell which $(CC)) $(THISFILE)
-	if [ "$(shell printf "$(shell $(CC) -dumpversion)\n8" | sort -Vr | head -1)" = 8 ]; then \
-		$(COMPILE.cpp) -o gcc9+support.o gcc9+support.c ;\
+$(MAKEFILE_LINK): $(shell which $(CXX)) $(THISFILE)
```

Comment:
The dependency on `CXX` seems a bit unusual here, it should probably depend on the `gcc9+support.cpp` file here also.

Reply:
The thought behind this is that this check will be rerun when the system(/compiler) is updated. So if previously it was on gcc 7 it will then switch to the gcc8+ Makefile. I can add a comment for that.
It should also depend on the `gcc9+support.cpp` file, yeah.